drivers/storage/portworx/component/telemetry.go

```diff
@@ -2,6 +2,7 @@ package component
 
 import (
 	"context"
+	cryptoTls "crypto/tls"
```

Comment:
That is because we have a custom tls type defined in this package
https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/component/tls.go#L24
so I have to distinguish

Reply:
Ahh.. that explains it -- a poor name for the operator component